src/index.js

```diff
@@ -78,7 +78,90 @@ async function listPackages (ghToken, repoName, repoOwner, packageName, packageC
     }
   }
   )
-  return dockerPkg.repository.packages.edges[0].node.versions.edges
+
+  if (dockerPkg.repository.packages.edges[0].node.versions.edges === undefined) {
+    throw 'Error: Repository has no containers!';
+  }
+
+
+  console.log('package list')
```

Comment:
if we want to keep these here we should have some debug flag that would cause these to print out, it would be an option the main function that would default to "false" (or something like that)